tree-sitter-dtd/grammar.js

```diff
@@ -12,6 +12,11 @@ const O = optional;
 module.exports = grammar({
   name: 'dtd',
 
+  externals: $ => [
+    $.PITarget,
+    $.PIContent,
```

Comment:
This should be hidden and doesn't have to follow the standard naming scheme since it's not part of it.

```suggestion
    $._pi_content,
```